src/Data/Codec/Argonaut/Common.purs

```diff
@@ -1,25 +1,30 @@
 module Data.Codec.Argonaut.Common
-  ( nonEmptyString
-  , nonEmptyArray
-  , maybe
-  , tuple
-  , either
+  ( either
+  , foreignObject
   , list
-  , nonEmptyList
   , map
-  , set
-  , nonEmptySet
-  , foreignObject
+  , maybe
   , module Data.Codec.Argonaut
+  , nonEmptyArray
+  , nonEmptyList
+  , nonEmptySet
+  , nonEmptyString
+  , set
+  , strMap
+  , tuple
```

Comment:
Sorry, this is coming from the autoformat, which has sorted in IDE order. Let me know if you want it reverted!